code/game/turfs/simulated/floor/beach.dm

```diff
@@ -0,0 +1,71 @@
+/turf/simulated/floor/beach/away
```

Comment:
Still got away in the path

Reply:
Yeah, I know its a PITA but this should be part of a wider water/beach turf refactor IMO, there's a lot of crap piled onto these and these turfs work in a really dumb way a lot of the time. Not to mention there is a non-away turf path for these that does something different... go figure.